components/automate-ui/src/app/modules/infra-proxy/migration-slider/migration-slider.component.html

```diff
@@ -73,14 +73,16 @@ <h3>You have the option to choose which users to migrate</h3>
                     chefInput
                     formControlName="{{user.automate_username}}"
                     type="text"
-                    [attr.disabled]= "!user.is_conflicting"
-                    (blur)="handleName($event, i)"
+                    attr.data-username="{{user.username}}"
+                    [attr.disabled]="!user.is_conflicting"
+                    (change)="onChangeEvent($event)"
+                    (blur)="handleName($event)"
                     autocomplete="off" />
                 </chef-td>
                 <chef-td class="last">
                   <p class="warning-msg"
-                   *ngIf="(user.connector === 'ldap') && user.is_conflicting">
-                   Warning. User already exists in Automate. This will overwrite this users IAM settings.
+                   *ngIf="(user.connector === 'LOCAL') && user.is_conflicting">
```

Comment:
We need to change this to 'local' in small caps, as this is getting changed at the backend.